### PR TITLE
Add documentation and testing for .containerignore

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test 
> /kind feature
> /kind flake
> /kind other

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: docs/buildah-bud.md [hunk 1]
```diff
@@ -803,22 +803,22 @@ are stored while pulling and pushing images.  Defaults to '/var/tmp'.
...
-If the file .dockerignore exists in the context directory, `buildah copy` reads
-its contents. Use `--ignorefile` flag to override .dockerignore path location.
-Buildah uses the content to exclude files and directories from the context
-directory, when executing COPY and ADD directives in the
-Containerfile/Dockerfile
+If the .containerignore/.dockerignore file exists in the context directory,
+`buildah copy` reads its contents. Use `--ignorefile` flag to override
+the ignore file path location. Buildah uses the content to exclude files and
+directories from the context directory, when executing COPY and ADD directives
+in the Containerfile/Dockerfile
```
Comment (vrothberg) [inline docs/buildah-bud.md:812]: What if both files exist in the context directory?  I assume that `.containerignore` will be used and the other is ignored?

**Review by vrothberg — COMMENTED** [review]
Just one question. Other than that LGTM

File: docs/buildah-bud.md (hunk 1 above)
Comment (rhatdan) [reply docs/buildah-bud.md:812]: Correct.

File: docs/buildah-bud.md [hunk 2]
```diff
@@ -803,22 +803,22 @@ are stored while pulling and pushing images.  Defaults to '/var/tmp'.
...
-### `.dockerignore`
+### `.containerignore`/`.dockerignore`
 
-If the file .dockerignore exists in the context directory, `buildah copy` reads
-its contents. Use `--ignorefile` flag to override .dockerignore path location.
-Buildah uses the content to exclude files and directories from the context
-directory, when executing COPY and ADD directives in the
-Containerfile/Dockerfile
+If the .containerignore/.dockerignore file exists in the context directory,
+`buildah copy` reads its contents. Use `--ignorefile` flag to override
```
Comment (TomSweeneyRedHat) [inline docs/buildah-bud.md:809]:
```suggestion
`buildah copy` reads its contents. Use the `--ignorefile` flag to override
```

**Comment by TomSweeneyRedHat** [comment]
One small man page nit then LGTM

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Pull request merged**